apps/paper/src/TypeGPU/GradientTiles/gradientWgsl.ts

```diff
@@ -0,0 +1,46 @@
+export const vertWGSL = `
+struct Output {
+  @builtin(position) posOut: vec4f,
+  @location(0) uvOut: vec2f,
```

Comment:
In this case, these exist on the Output struct, so we can just call them `pos` and `uv`.
```suggestion
  @builtin(position) posOut: vec4f,
  @location(0) uvOut: vec2f,
```